FV3/conf/configure.fv3.linux_gnu

```diff
@@ -66,14 +65,14 @@ CFLAGS += -D__IFC
 CFLAGS_OPT = -O2 
 CFLAGS_REPRO = -O2 
 CFLAGS_OPENMP = -fopenmp
-CFLAGS_DEBUG = -O0 -g 
+CFLAGS_DEBUG = -O0 -g
 
-LDFLAGS :=
+LDFLAGS := -L${FMS_DIR}/lib -L${ESMF_DIR}/lib
 LDFLAGS_OPENMP := -fopenmp
 LDFLAGS_VERBOSE := -Wl,-V,--verbose,-cref,-M
 
 # start with blank LIBS
-LIBS := -lgfortran
+LIBS := -lgfortran -lFMS -lesmf
```

Comment:
How was FMS and esmf included before?

Reply:
Both libraries were statically linked and explicitly mentioned in configure.fv3.gnu_docker previously:

The ESMF library was expected to be at: ${ESMF_DIR}/lib/libO3/Linux.gfortran.64.mpiuni.default/. 
This assumed that one always built ESMF with optimization level 3, using the GNU compilers and providing no MPI support.  Now, the new dockerfile specifies the ESMF library to be at ${ESMF_DIR}/lib.  This is more generic and gives more freedom to choose different optimization levels, MPI flavors, etc.

The FMS library was expected to be at: ${FMS_DIR}/libFMS/.libs.  Similarly to ESMF, the FMS libraries are now located at ${FMS_DIR}/lib

Reply:
Yeah, that was totally broken. Do you want to adapt it in configure.fv3.gnu_docker as well?